templates/dashd.conf.template

```diff
@@ -1,50 +1,57 @@
 # general
-
-devnet=evonet-6
-
 daemon=0  # leave this set to 0 for Docker
 logtimestamps=1
 maxconnections=256
-
-# optional indices (required for Insight)
-txindex=1
-addressindex=1
-timestampindex=1
-spentindex=1
-
-# ZeroMQ notifications (required for Insight)
-zmqpubrawtx=tcp://0.0.0.0:29998
-zmqpubrawtxlock=tcp://0.0.0.0:29998
-zmqpubhashblock=tcp://0.0.0.0:29998
+printtoconsole=1
 
 # JSONRPC
 server=1
-rpcport=20002
-rpcuser=dashrpc
-rpcpassword=password
+rpcuser={{=it.core.rpc.user}}
+rpcpassword={{=it.core.rpc.password}}
 
-rpcallowip=127.0.0.0
+rpcallowip=127.0.0.1
 rpcallowip=172.16.0.0/12
-rpcallowip=172.27.0.0/16
 rpcallowip=192.168.0.0/16
 
 rpcworkqueue=64
 rpcthreads=16
 
 # external network
 listen=1
-bind=0.0.0.0
 dnsseed=0
 allowprivatenet=0
 
-printtoconsole=1
+# need this twice to support 0.15 and 0.16 conf files
+port={{=it.core.p2p.port}}
+rpcport={{=it.core.rpc.port}}
 
+
+{{? it.network.name === 'testnet'}}
+{{??}}
+# optional indices (required for Insight)
+txindex=1
+addressindex=1
+timestampindex=1
+spentindex=1
+
+# ZeroMQ notifications (required for Insight)
+zmqpubrawtx=tcp://0.0.0.0:{{=it.core.zmq.port}}
+zmqpubrawtxlock=tcp://0.0.0.0:{{=it.core.zmq.port}}
+zmqpubhashblock=tcp://0.0.0.0:{{=it.core.zmq.port}}
+{{?}}
+
+
+{{? it.network.name === 'local'}}
+regtest=1
+[regtest]
+{{?? it.network.name === 'evonet'}}
+devnet={{=it.network.name}}-{{=it.network.version}}
+[devnet]
 addnode=seed-1.evonet.networks.dash.org:20001
```

Comment:
They should be configurable as well.

Reply:
What do you mean by "they"?

Reply:
Sorry, I meant `addnode` options should be configurable as well.

Reply:
OK, added `seeds` and `sporkAddr` to config.